xds/internal/xdsclient/bootstrap/bootstrap.go

```diff
@@ -151,6 +165,10 @@ type ServerConfig struct {
 	// when a resource is deleted, nor will it remove the existing resource value
 	// from its cache.
 	IgnoreResourceDeletion bool
+
+	// Cleanups are called when the xDS client for this server is closed. Allows
+	// cleaning up resources created specifically for the xDS client.
```

Comment:
s/created specifically for the xDS client/created specifically for this ServerConfig/ ?

Reply:
Done.